.ci/pytorch/test.sh

```diff
@@ -991,8 +991,25 @@ test_docs_test() {
 
 test_executorch() {
   # Test torchgen generated code for Executorch.
-  echo "Testing Executorch op registration"
+  echo "Testing ExecuTorch op registration"
   "$BUILD_BIN_DIR"/test_edge_op_registration
+
+  pushd /executorch
+
+  echo "Install torchvision and torchaudio"
+  # TODO(huydhn): Switch this to the pinned commits on ExecuTorch once they are
+  # there.  These libraries need to be built here, and not part of the Docker
+  # image because they require the target version of torch to be installed first
+  pip_install --no-use-pep517 --user "git+https://github.com/pytorch/audio.git"
+  pip_install --no-use-pep517 --user "git+https://github.com/pytorch/vision.git"
+
+  echo "Run ExecuTorch regression tests for some models"
+  # NB: This is a sample model, more can be added here
+  export PYTHON_EXECUTABLE=python
+  # shellcheck disable=SC1091
+  source .ci/scripts/test.sh mv3 cmake xnnpack-quantization-delegation ''
```

Comment:
I'm a bit confused which torch is testing against. IIUC, it should be using the commit of a PR and test if any changes will break ExecuTorch, so the pinned torch is a commit of a PR and torchaudio and torchvision are built on same commit from source. Is it what we doing? I failed to see where the commit of a PR is set and where torch libraries are built

Reply:
This one uses:

* PyTorch build from the PR in `linux-jammy-py3-clang12-executorch-build` step.  Basically, the build step builds PyTorch and the build artifacts are copied over to the test job (this script).  So by the time ET test is run, it's guaranteed that PyTorch from the PR is used.  You can see the installation happening here https://github.com/pytorch/pytorch/actions/runs/6837644212/job/18594107600#step:16:132
* It's currently using latest main commit from vision and audio.  The command `pip_install --no-use-pep517 --user "git+https://github.com/pytorch/vision.git` does it.  Once we have pinned commits for vision and audio on Executorch, we can use them instead, thus the TODO part above